artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -2712,6 +2713,11 @@ private Message makeCopy(final MessageReference ref, final boolean expiry) throw
    private Message makeCopy(final MessageReference ref,
                             final boolean expiry,
                             final boolean copyOriginalHeaders) throws Exception {
+      if (ref == null) {
+         ActiveMQServerLogger.LOGGER.nullRefMessage();
+         throw new ActiveMQNullRefException("Reference to message is null");
```

Comment:
Was this an accident? WEren't you supposed to use NULL_REF.createException here?